microshift_support/microshift-etcd.adoc

```diff
@@ -0,0 +1,15 @@
+:_content-type: ASSEMBLY
+[id="microshift-etcd"]
+= MicroShift etcd
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-etcd
+
+toc::[]
+
+:FeatureName: MicroShift
```

Comment:
I'm noticing that throughout the docs we have the tech preview snippet in different places. Sometimes its at the beginning of the assembly and sometimes its after the intro. I'm wondering what you think looks better?

Reply:
Good question @kelbrown20 ! Lets mention it in the docs call in a bit.